godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml

```diff
@@ -1,6 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: 'joint_trajectory_action'
+    action_ns: ''
```

Comment:
Jorge:
Is this the warning you are talking about, because we didn't set the
action_ns.

or something else, like maybe GOAL_COMPLETION_BUFFER is not long enough.

[ INFO] [1459201082.801943579]: Handling new simulation service request
[ WARN] [1459201094.921052790]: Robot Simulator did not successfully
complete trajectory



On Tue, Mar 8, 2016 at 9:37 AM, Jorge Nicho <notifications@github.com>
wrote:

> In
> godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml
> <https://github.com/swri-robotics/godel/pull/1#discussion_r55376276>:
>
> > @@ -1,6 +1,5 @@
> >  controller_list:
> > -  - name: ""
> > -    action_ns: joint_trajectory_action
> > +  - name: 'joint_trajectory_action'
> > +    action_ns: ''
>
> I thought it was necessary to specify the *action_ns* . Do you get any
> warnings related to actions not being found?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swri-robotics/godel/pull/1/files#r55376276>.
>